docs/architecture/adr-044-lite-client-with-weak-subjectivity.md

```diff
@@ -0,0 +1,70 @@
+# Tendermint Lite Client with Weak Subjectivity ADR
+
+Changelog:
+- 13-07-2019: Initial Draft
+## Context
+The concept of light clients was introduced in the Bitcoin white paper. It describes a watcher of distributed consensus process that only validates the consensus algorithm and not the state machine transactions within.
+
+Tendermint light clients allow light weight devices and other blockchains to efficiently verify the consensus of a Tendermint blockchain. This forms the basic of safe and efficient state synchronization for new network nodes and InterBlockchain Communication.
+
+In a network that is expected to reliably punish validators for misbehavior through punishments or where the validator set is largely trusted and changes infrequently, clients can take advantage of this assumption to safely synchronize a lite client without downloading the intervening headers.
+
+Light clients (and full nodes) operating in the Proof Of Stake context need a trusted block height from a trusted source that is no older than 1 unbending window. This is called “weak subjectivity”
+
+Currently, Tendermint provides a lite client implementation in the [tendermint/lite](https://github.com/tendermint/tendermint/tree/master/lite) directory. This lite client implements a bisecting algorithm that tries to use a binary search to find the minimum number of block headers where the validator set voting power changes are less than <1/3rd. This interface does not support weak subjectivity at this time. The Cosmos SDK also does not support counterfactual slashing or does the lite client have any capacity to report evidence making these systems theoretically unsafe.
+
+## Decision 
+
+### The Weak Subjectivity Interface.
+
+Weak Subjectivity is required in Proof of Stake blockchains because it is costless for an attack to buy up voting keys that are no longer bonded and fork the network at some point it’s history. See Vitalik’s post at [Proof of Stake: How I Learned to Love Weak Subjectivity](https://blog.ethereum.org/2014/11/25/proof-stake-learned-love-weak-subjectivity/)
+
+This requires providing a user interface for when new nodes connect to the network or when a node that has been offline for longer than the unbending period connects to the network. 
+
+Specifically, the node node needs to initialize the following structure before synching from user input. The expectation is the user will get this information from a trusted source like a validator, https://cosmos.network or other.  A more user friendly solution with trust tradeoff is that we establish an https based protocol with a default end point that populates this information. 
```

Comment:
I would note that an on-chain registry of roots-of-trust (e.g. on the Cosmos Hub) seems likely in the future.